### DISCUSSION TITLE
允许安装在子服的MiraiMC插件调用安装在Bungee的MiraiMC

### DISCUSSION THREAD

**Discussion by LittleChest** [DreamVoid/MiraiMC#329] [建议 / Ideas]
我不知道目前有没有这个功能，来讨论问一下
区分子服由插件自行实现（例如Chat2QQ提供自定义聊天格式、进退服消息，可以由用户在前面加前缀区分）
可以由MiraiMC下发一个数据包（具体实现我也不太懂）来让插件检测MiraiMC
Bungee（安装MiraiMC）
    - Paper（安装Chat2QQ）
    - Paper（安装Chat2QQ）
类似于上面这种

**Comment by DreamVoid** [comment]
正在考虑添加，但是现在的开发重心是把我写的屎山代码优化一下。
